google/cloud/storage/examples/storage_object_acl_samples.cc

```diff
@@ -260,6 +260,83 @@ void PatchObjectAclNoRead(gcs::Client client, int& argc, char* argv[]) {
   (std::move(client), bucket_name, object_name, entity, role);
 }
 
+void AddObjectOwner(gcs::Client client, int& argc, char* argv[]) {
+  if (argc != 4) {
+    throw Usage{"add-object-owner <bucket-name> <object-name> <entity>"};
+  }
+  auto bucket_name = ConsumeArg(argc, argv);
+  auto object_name = ConsumeArg(argc, argv);
+  auto entity = ConsumeArg(argc, argv);
+  //! [add file owner] [START storage_add_file_owner]
+  namespace gcs = google::cloud::storage;
+  using ::google::cloud::StatusOr;
+  [](gcs::Client client, std::string bucket_name, std::string object_name,
+     std::string entity) {
+    StatusOr<gcs::ObjectAccessControl> patched_acl =
+        client.CreateObjectAcl(bucket_name, object_name, entity,
+                               gcs::ObjectAccessControl::ROLE_OWNER());
+
+    if (!patched_acl) {
+      throw std::runtime_error(patched_acl.status().message());
+    }
+
+    std::cout << "ACL entry for " << patched_acl->entity() << " in object "
+              << patched_acl->object() << " in bucket " << patched_acl->bucket()
+              << " is now " << *patched_acl << "\n";
+  }
+  //! [add file owner] [END storage_add_file_owner]
+  (std::move(client), bucket_name, object_name, entity);
+}
+
+void RemoveObjectOwner(gcs::Client client, int& argc, char* argv[]) {
```

Comment:
When I ran this sample manually, it had a noticeable delay. Performance improvements welcome.